src/twisted/newsfragments/9410.bugfix

```diff
@@ -0,0 +1 @@
+write() to disconnected twisted.web.http.Request will no longer raise AttributeError.
```

Comment:
According to the [style guide](https://twistedmatrix.com/trac/wiki/ReviewProcess#Newsfiles), "twisted.web.http.Request.write after the channel is disconnected will no longer raise AttributeError.", I think.  (Suggestions on how to make that information more easily discoverable welcome)

Reply:
fixed in 4d7733c